feature-profile/src/main/java/cm/aptoide/pt/profile/presentation/ProfileView.kt

```diff
@@ -509,7 +499,7 @@ private fun AptoideSocialsIcon(
         .clip(CircleShape)
         .border(
           width = 1.dp,
-          color = grey,
+          color = greyMedium,
```

Comment:
The border depends on the app theme. It should be grey on light theme and greyMedium on dark theme